compiler/deepdive-default.conf

```diff
@@ -12,7 +12,7 @@ deepdive {
   }
 
   sampler.sampler_cmd: "sampler-dw"
-  sampler.sampler_args: "-l 1000 -s 1 -i 1000 --alpha 0.01"
+  sampler.sampler_args: "-l 1000 -s 1 -i 1000 --alpha 0.01 --sample_evidence"
```

Comment:
@feiranwang Good catch!  @thodrek This is actually the default settings we updated, not just our example.

We've seen too many people bitten by this default setting, and enough is enough.  Whoever that's rigorous enough wanting to exactly evaluate the system's performance won't be misled by the extra inference results.  So I insist on turning this option on by default.  However, I'm all for adding a clear way to prevent misinterpretation, so let's discuss what can be the best additional output, e.g., what extra view can we create and/or should we compute two numbers that excludes/includes holdout?